package.json

```diff
@@ -13,6 +13,7 @@
   "dependencies": {
     "axios": "^0.19.2",
     "express": "^4.17.1",
+    "jquery": "^3.5.1",
```

Comment:
jquery?!?!? uh ohh 😆

Reply:
yeah unfortunately it's the only way to test html DOM elements with JEST

Reply:
good to know!